x-pack/plugins/synthetics/public/apps/synthetics/components/test_now_mode/browser/browser_test_results.tsx

```diff
@@ -120,6 +120,7 @@ export const BrowserTestRunResult = ({ expectPings, onDone, testRunId }: Props)
                   compressed={true}
                   testNowMode={true}
                   showLastSuccessful={false}
+                  allStepsLoaded={stepLoadingInProgress}
```

Comment:
@shahzad31 does this guarantee that screenshot info is available? It's possible for all the steps to show up but the `screenshot_ref` data not to be queryable yet in Elasticsearch?

Reply:
i think @awahab07 originally added this logic. I just wanted to see  and it seems to be working. I am fine with both approaches though. I think we can keep both.

Reply:
ok aplogies, i thought it was working. So both can't go alongside. let's keep the exponential. please revert my change. Or i can do it tomorrow. Otherwise PR looks good.

Reply:
No problem, if just that little change worked I'd have said we could undo the solution I wrote and use that one instead!